### ISSUE TITLE
Cannot delete concept terms or Education Level Types using keyboard only

### ISSUE THREAD

**Issue by eddavies** [cassproject/cass-editor#73]
As title describes.

**Comment by eddavies** [comment]
Also using alt-left to go back/cancel the search makes it so tab does not select any of the fields within the framework, only the url and browser specific buttons.

**Comment by eddavies** [comment]
This is actually 3 issues, should I make one for each?

**Comment by eddavies** [comment]
Also potentially a fourth, are concept terms and education level types supposed to be viewed in view/download form?

**Comment by miledivovic** [comment]
Fixed in 76d03c3

**Comment by eddavies** [comment]
fix verified

**Comment by Lomilar** [comment]
<button> can have display issues which don't work well with float:right. Can we pick another? Or set tabindex="0" on the <a> tag?

**Comment by Lomilar** [comment]
![image](https://user-images.githubusercontent.com/337692/33503367-5131613c-d698-11e7-9288-1c785b37fe19.png)

**Comment by Lomilar** [comment]
... he says, being able to make those changes himself. OK to test.

**Comment by eddavies** [comment]
The X is selectable, but clicking enter or spacebar does not delete the term.

**Comment by miledivovic** [comment]
Fixed in e50783b

**Comment by eddavies** [comment]
fix verified
